x/launch/client/cli/tx_create_chain.go

```diff
@@ -21,9 +22,9 @@ const (
 
 func CmdCreateChain() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "create-chain [genesis-chain-id] [source-url] [source-hash]",
+		Use:   "create-chain [genesis-chain-id] [source-url] [source-hash] [metadata]",
```

Comment:
I think we can set metadata as a flag and not a required argument in the CLI

We can also think of a flag like `--metadata-file` if we would eventually provide the value as a json but I would say let's see how we use this field